router/core/graphql_prehandler.go

```diff
@@ -614,6 +616,13 @@ func (h *PreHandler) handleOperation(req *http.Request, buf *bytes.Buffer, httpO
 		return nil, err
 	}
 
+	if h.queryPlansLoggingEnabled {
+		switch p := opContext.preparedPlan.preparedPlan.(type) {
+		case *plan.SynchronousResponsePlan:
+			h.log.Info(fmt.Sprintf("Query Plan:\n%s", p.Response.Fetches.QueryPlan().PrettyPrint()))
```

Comment:
Please use zap log fields

Reply:
e.g.
```go
h.log.Info("Query Plan representation", zap.String("query_plan", p.Response.Fetches.QueryPlan().PrettyPrint()))
```

Reply:
it won't be readable in dev mode

anything structure getting printed escaped

Reply:
probably I could log with a field when dev mode disabled?

Reply:
Got it! I like your idea. Let's log it as a JSON field in production mode and in development as follow:

`s.logger.Sugar().Debugf("Query Plan:\n%s", ...)`